docs/book/src/user/troubleshooting.md

```diff
@@ -68,15 +68,15 @@ misconfigured or there might be problems with the network. The following steps d
 be troubleshooted systematically.
 
 1. Access the Node via ssh.
-1. Take a look at cloud-init logs via `less /var/log/cloud-init-output.log` or `journalctl -u cloud-init --since "1 day ago"`.
+2. Take a look at cloud-init logs via `less /var/log/cloud-init-output.log` or `journalctl -u cloud-init --since "1 day ago"`.
    (Note: cloud-init persists logs of the commands it executes (like kubeadm) only after they have returned.)
-1. It might also be helpful to take a look at `journalctl --since "1 day ago"`.
-1. If you see that kubeadm times out waiting for the static Pods to come up, take a look at:
+3. It might also be helpful to take a look at `journalctl --since "1 day ago"`.
+4. If you see that kubeadm times out waiting for the static Pods to come up, take a look at:
    1. containerd: `crictl ps -a`, `crictl logs`, `journalctl -u containerd`
    1. Kubelet: `journalctl -u kubelet --since "1 day ago"`
```

Comment:
```suggestion
   2. Kubelet: `journalctl -u kubelet --since "1 day ago"`
```

Technically just using 1. everywhere also works > https://www.markdownguide.org/basic-syntax/#ordered-lists

(but I also get a warning in Intellij. I'm fine either way, definitely nice if the source looks more similar to the rendered output)

Reply:
Right, I wanted to correct the flag name, but saw this alongside. 🙂